controllers/front/AttachmentController.php

```diff
@@ -32,7 +32,7 @@ public function postProcess()
             Tools::redirect('index.php');
         }
 
-        Hook::exec('actionDownloadAttachment', ['attachment' => &$a]);
+        Hook::exec('actionDownloadAttachment', ['attachment' => $attachment]);
```

Comment:
```suggestion
        Hook::exec('actionDownloadAttachment', ['attachment' => &$attachment]);
```

I think you could leave this as is to allow developers modify this var

Reply:
Fixed